Source/VCExpr/SimplifyLikeLineariser.cs

```diff
@@ -392,6 +392,7 @@ public class SimplifyLikeExprLineariser : IVCExprVisitor<bool, LineariserOptions
     internal const string floatGeqName = "floatGeq";
     internal const string floatGtName = "floatGt";
     internal const string floatEqName = "floatEq";
+    internal const string floatNeqName = "floatEq";
```

Comment:
This seems to be a copy-paste typo, no? You are using the same string (`floatEq`) in both cases. Is that expected?